Jobsway2go.Application/Services/UserService.cs

```diff
@@ -45,7 +47,7 @@ public async Task<bool> DeleteUser(string userId)
             }
             else
                 return false;
-            //Nuk jomm tu dit qa me bo return a naj action to Index ??
+
             return false;
         }
```

Comment:
Does this one save to context?

Reply:
Hale skemi Dbcontext qe me e rujt , kur t krijohet duhet me e shtu tani. Te PR e fundit nuk osht ky koment, osht edhe action method te controlleri.